CHANGELOG.md

```
@@ -721,31 +710,27 @@ See the new [Install documentation](https://docs.espressif.com/projects/vscode-e
+## [0.5.0](https://github.com/espressif/vscode-esp-idf-extension/releases/tag/v0.5.0) - 2020-09-24
 ### ESP-ADF & ESP-MDF
 
 - Added support for ESP-ADF and ESP-MDF
 - Fixed some bugs related to [Device Baud Rate](https://github.com/espressif/vscode-esp-idf-extension/pull/166), [Webview Background Color](https://github.com/espressif/vscode-esp-idf-extension/pull/166), [Menuconfig](https://github.com/espressif/vscode-esp-idf-extension/pull/166)
 
-## [0.4.0](https://github.com/espressif/vscode-esp-idf-extension/releases/tag/v0.4.0)
-
+## [0.4.0](https://github.com/espressif/vscode-esp-idf-extension/releases/tag/v0.4.0) - 2020-07-23
 ### System View Tracing Viewer, Search IDF Documentation, etc.
 
 - Add System View Tracing [support]() inside vscode, which will enable you to easily view the timeline, events stream, etc., of of existing heap tracing (.svdat) file
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)